src/gol.test.js

```diff
@@ -0,0 +1,161 @@
+import next, { countLiveNeighbours } from './gol';
+
+describe('countLiveNeighbours', () => {
+  test('return 8 when all cells are live', () => {
+    const input = [
+      [1, 1, 1], 
+      [1, 1, 1], 
+      [1, 1, 1],
+    ];
+    const expectedOutput = 8;
+    expect(countLiveNeighbours(input, 1, 1)).toBe(expectedOutput);
+  });
+  test('return 3 when all cells around corner are live', () => {
+    const input = [
+      [0, 1, 1],
+      [1, 1, 1],
+      [1, 1, 1],
+    ];
+    const expectedOutput = 3;
+    expect(countLiveNeighbours(input, 0, 0)).toBe(expectedOutput);
+  });
+  test('return 3 when all cells around corner are live', () => {
```

Comment:
I don't think that use same 'description' for multiple tests is a good practice.